pkg/metrics/metrics.go

```diff
@@ -22,12 +24,17 @@ type Metrics struct {
 	skipped prometheus.Counter
 }
 
-// RegisterScan registers metrics for an executed scan
+// Register registers metrics for an executed scan
 func (metrics *Metrics) Register(metric *Metric) {
 	metrics.channel <- metric
 }
 
+// New creates a new metrics handler if none exists, otherwise returns the existing one
 func New() *Metrics {
+	if metrics != nil {
+		return metrics
+	}
+
```

Comment:
That name is a bit unintuitive, perhaps `DefaultMetrics()` instead, like [net/http.DefaultClient](https://golang.org/pkg/net/http/#DefaultClient)